### DISCUSSION TITLE
Removing large files fails under Windows

### DISCUSSION THREAD

**Discussion by stell** [gitextensions/gitextensions#11093] [General]
### Environment

- Git Extensions 4.1.0.16698
- Git 2.41.0.windows.1
- Microsoft Windows NT 10.0.19045.0
- .NET 6.0.19
- Portable: False
- Microsoft.WindowsDesktop.App Versions

### Issue description

Using the removing large files plugin under Windows I tried different approaches for the bash script to not fail calling Git.

I tried different paths to git, but i don't think they are the problem.

```
>for /F %a IN ('"C:\Program Files\Git\bin\git.exe" for-each-ref --format "%^(refname^)" refs/original/') DO "C:\Program Files\Git\bin\git.exe" update-ref -d %
Command "C:\Program"" for-each-ref --format "%(refname)" is wrong or could not be found.
```
I also tried this:

`Command "C:\PROGRA~1\Git\bin\git.exe" for-each-ref --format "%(refname)" is wrong or could not be found.`

without success.

Looks like generating a prompt with single quotes instead of double quotes is the problem.

### Steps to reproduce

Simply try to remove a file under windows.

### Did this work in previous version of GitExtensions?

don't know.

**Comment by gerhardol** [comment]
This is not a question about Git Extensions but about Git, wrong forum.

You can add Git to your PATH and just type Git.
For the full path, the quoting depends on the shell you use. A '\' escapes a space in bash.
